lib/lunchbot/lunchbot_data/office_lunch_order.ex

```diff
@@ -16,4 +17,25 @@ defmodule Lunchbot.LunchbotData.OfficeLunchOrder do
     |> cast(attrs, [:day, :office_id])
     |> validate_required([:day, :office_id])
   end
+
+  def get_lunch_order_for_office(office \\ "Providence", date \\ Date.utc_today()) do
+    filter_lunch_orders_to_today_only = from(o in OfficeLunchOrder, where: o.day == ^date)
```

Comment:
When we reference `OfficeLunchOrder`, even when we're inside the schema file itself, I think we still need to reference it fully with an alias or like `Lunchbot.LunchbotData.OfficeLunchOrder`

Reply:
Whoops, thats from some funky rebasing I did earlier, good catch